Doc/library/os.rst

```diff
@@ -3574,9 +3575,9 @@ written in Python, such as a mail server's external command delivery program.
 
    If *mode* is :const:`P_NOWAIT`, this function returns the process id of the new
    process; if *mode* is :const:`P_WAIT`, returns the process's exit code if it
-   exits normally, or ``-signal``, where *signal* is the signal that killed the
-   process.  On Windows, the process id will actually be the process handle, so can
-   be used with the :func:`waitpid` function.
+   exits normally, or ``-signal`` on non-VxWorks platform, where *signal* is the 
+   signal that killed the process.  On Windows, the process id will actually be the 
+   process handle, so can be used with the :func:`waitpid` function.
 
```

Comment:
Ok, now I better understand the subtle difference. I suggest to describe it on a separated paragraph to not confuse users on other platforms. Something like:

On VxWorks, if *mode* is :const:`P_WAIT` and the process is killed by a signal, return -1.

Reply:
Oh, you made the change before I asked for it :-) Thanks!